### ISSUE TITLE
Constant high gpu usage

### ISSUE THREAD

**Issue by f1Lthz4** [JekiTheMonkey/KPS#23]
App uses 10-20% of 4080 doesn't matter if it draws anything or not
![screenshot](https://i.imgur.com/Zo0pCDO.png)

**Comment by Tonetfal** [comment]
Yeah, it's an undefined behavior a very small amount of users have. Usually, if someone has it, it's always the case for them. I have never had any issue with the GPU usage, so I couldn't really track down the issue just by the fact that it is there for some people. Sometimes it's the settings window that takes that much of GPU for no reason, so you can try to close it; again, it's something a few people experience. Unfortunately, I can't help with it.

If you're a programmer, and you want to find out and fix the issue, I would suggest trying to setup the environment for the program, i.e. installing SFML 2.5.1 along Visual Studio, and check whether the Tick is not running more than 60 times per second. It shouldn't, but if does, it means that most likely there's an issue with underlying libs/frameworks the program relies on.
